sdv/metadata/multi_table.py

```diff
@@ -516,14 +516,16 @@ def add_table(self, table_name):
 
         self.tables[table_name] = SingleTableMetadata()
 
-    def visualize(self, show_table_details=True, show_relationship_labels=True,
+    def visualize(self, show_table_details='full', show_relationship_labels=True,
                   output_filepath=None):
         """Create a visualization of the multi-table dataset.
 
         Args:
-            show_table_details (bool):
-                If True, the column names, primary and foreign keys are all shown along with the
-                table names. If False, only the table names are shown. Defaults to True.
+            show_table_details (str or None):
+                If 'full', show the column names, primary and foreign keys are all shown along with
+                the table names. If 'summarized' primary and foreign keys are shown and a count
+                of the different sdtypes is shown, if None only the table names are shown. Defaults
```

Comment:
minor: I'd add a new sentence for the final case like the others (ie. "shown. If None, ...")